tests/test_photon_ops.py

```diff
@@ -217,8 +221,8 @@ def test_lsst_diffraction_shows_field_rotation() -> None:
         / np.cos(lsst_diffraction.altitude / 180.0 * np.pi)
     )
 
-    np.testing.assert_almost_equal(
-        cross_rot_angle_1 - cross_rot_angle_0, expected_angle_difference, decimal=2
+    np.testing.assert_allclose(
+        cross_rot_angle_1 - cross_rot_angle_0, expected_angle_difference, rtol=0.13
```

Comment:
I think this is probably fine, although 0.13 is still a pretty large tolerance.  But probably worth some developer commentary on why you chose this value.  I.e. give the numerical results and speculate why it's not closer than this.

One possibility is that it's due to the rotation angle not being quite constant even over this short time when it's so close to zenith.  Maybe worth trying another position that's a little farther from zenith, so that the rotation is still easily measurable, but maybe would be more uniform and thus agree with the prediction better.

Reply:
So, the fact that the rotation angular rate is not constant over the time interval actually plays a role. However, taking this into account increases the discrepancy.
The actual reason just was the ridiculous small amount of spike photons (36 only).
I increased the amount of photons by a factor of 100 now and this seems to be much more accurate.
I now directly compared against rotation angle instead of the rate.

In exchange, I improved the test relating the angle to its rate, by using numerical integration (test_diffraction.py)